src/api/app/models/concerns/workflow_run_gitea_payload.rb

```diff
@@ -54,6 +57,22 @@ def gitea_pull_request?
     scm_vendor == 'gitea' && hook_event == 'pull_request'
   end
 
+  def gitea_ping?
+    scm_vendor == 'gitea' && hook_event == 'ping'
+  end
+
+  def gitea_supported_event?
+    scm_vendor == 'gitea' && ALLOWED_GITHUB_EVENTS.include?(hook_event)
```

Comment:
`ALLOWED_GITHUB_EVENTS`?

Reply:
Upps, yes thats the wrong one... still works because it has similar content... I will correct it

Reply:
corrected it